### DISCUSSION TITLE
How to enable FLEXRIC in eNB

### DISCUSSION THREAD

**Discussion by altis82** [srsran/srsRAN_Project#300] [General Help]
Hi everyone. Currently I am working on srsRAN and usrp b210. It works very well to connect with phones. In the document, I see that we can enable gNB with FlexRIC but could we do that with eNB? Since I could do that with OAI but not with srsRAN. 
Thanks for support

**Comment by brendan-mcauliffe** [comment]
Hi @altis82, the eNB does have a very basic e2 implementation. However, there is no tutorial for connecting the eNB to a RIC. We strongly recommend doing any work relating to RICs with the latest version of srsRAN Project.
